lib/media/media_source_engine.js

```diff
@@ -1415,3 +1482,15 @@ shaka.media.MediaSourceEngine.RAW_FORMATS = [
   'audio/ec3',
   'audio/mpeg',
 ];
+
+/**
+ * Get whether or not the engine supports codec switching.
+ *
+ * @return {boolean}
+ */
+shaka.media.MediaSourceEngine.isChangeTypeSupported = () => {
```

Comment:
Please rebase and include this method in https://github.com/shaka-project/shaka-player/blob/main/lib/media/media_source_capabilities.js Thanks!